src/frontend/next/public/logo.svg

```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="640" height="640">
```

Comment:
I think this is happening in #1481, cc @chrispinkney.  We should probably merge that first, and then use it here.  @jossiey, can you add a review to that PR for me?

Reply:
woohoo #1481 just got merged so yeah this file can be removed

Reply:
So, I am removing the public/logo.svg completely?

Reply:
@chrispinkney ^^

Reply:
AHH, sorry for the delay, I didn't even notice this. Yes, remove `logo.svg`:  If you rebase your current progress with master the logo will be created since it's already landed in the project.

Reply:
Done (just marking for myself, you can ignore that)